Packs/Hoxhunt/Integrations/HoxhuntV2/HoxhuntV2.py

```diff
@@ -0,0 +1,855 @@
+        }
+
+    def set_incident_soc_classification(self, incident_id: str | None, classification: str | None):
+        return self.query(MUTATIONS['SetIncidentSocClassification'], {
+            'incidentId': incident_id,
+            'classification': classification
+        })
+
+    def update_incident_state(self, incident_id: str | None, state: str | None):
+        return self.query(MUTATIONS['UpdateIncidentState'], {
+            'incidentId': incident_id,
+            'state': state
+        })
+
+    def get_current_user(self):
+        return self.query(QUERIES['GetCurrentUser'])
+
+    def get_incident_by_id(self, incident_id: str, last_update: str):
+        results = self.query(QUERIES['GetIncidentById'], {'lastUpdate': last_update, 'id': incident_id})
+        incidents = results.data.get('incidents')
+        if not incidents:
+            return {}
+
+        return incidents[0]
+
+    def get_modified_incidents(self, last_update: str):
+        return self.query(QUERIES['GetModifiedIncidents'], {"lastUpdate": last_update})
+
+    def update_changed_incident_fields(self, incident_id: str, delta: dict[str, Any]):
+        if delta.get('socClassification'):
+            self.set_incident_soc_classification(incident_id, delta.get('socClassification'))
+        if delta.get('isSensitive'):
+            self.set_incident_sensitive(incident_id, delta.get('isSensitive'))  # type: ignore
+
+    def handle_soc_feedback(self, incident_id: str, delta: dict[str, Any], incident_data: dict[str, Any]):
+        should_send_feedback = incident_data.get('shouldSendFeedback')
+        soc_message = incident_data.get('socMessage')
+        feedback_reported_at_limit = incident_data.get('feedbackReportedAtLimit')
+        updated_soc_classification = delta.get('socClassification')
+
+        # send feedback if enabled and if soc classification has changed (initial set or update)
+        if should_send_feedback and updated_soc_classification:
+            self.send_incident_soc_feedback(incident_id, soc_message, feedback_reported_at_limit)
+
+    def add_closing_incident_note(self, incident_id: str, incident_data: dict[str, Any]):
+        xsoar_close_note = incident_data.get("xsoarCloseNotes")
+        note = f"Incident closed via XSOAR{' - ' + xsoar_close_note if xsoar_close_note else ''}"
+        self.add_incident_note(incident_id, note)
+
+
+''' TYPES '''
+DemistoCommandReturnType = Union[CommandResults, PollResult, str, dict, BaseWidget,
+                                 list, GetMappingFieldsResponse, GetModifiedRemoteDataResponse, GetRemoteDataResponse]
+
+HoxCommandType = Callable[[Client, dict, dict], DemistoCommandReturnType]
+HoxNonReturningCommandType = Callable[[Client, dict, dict], None]
+
+''' FNS '''
+
+
+def create_output(results: dict[str, str], endpoint: str, key_field: str = '') -> CommandResults:
+    human_readable = tableToMarkdown('Hoxhunt results', results)
```

Comment:
Please use the `headers` and the `headerTransform` function arguments in order to became the headers more readable.
https://xsoar.pan.dev/docs/reference/api/common-server-python#tabletomarkdown